rel-eng/openshift-node.sysconfig

```diff
@@ -1,2 +1,5 @@
 ROLE="node"
 OPTIONS="--loglevel=0"
+# If running node on a separate host please rsync the admin directory ie:
+# rsync root@openshift-master:/var/lib/openshift/openshift.local.certificates/admin /var/lib/openshift/openshift.local.certificates/
+KUBECONFIG="/var/lib/openshift/openshift.local.certificates/admin/.kubeconfig"
```

Comment:
KUBECONFIG environment variable is, as far as I can tell, synonymous with --kubeconfig command line switch so it works as is. I was preferring to, whenever possible, use environment variables as it makes the unit file simpler but it's probably confusing to use both environment variables and command line switches.

I was thinking about lobbying for all command line args to work as environment variables as well which would allow us to simplify the unit files. What do you think?